src/app/pages/non-pass-finalisation/non-pass-finalisation.page.ts

```diff
@@ -370,4 +370,13 @@ export class NonPassFinalisationPage extends PracticeableBasePageComponent imple
       TestCategory.DM, TestCategory.D1M, TestCategory.DEM, TestCategory.D1EM,
     ]);
   };
+
+  DidTestComplete = (): boolean => {
+    if (this.activityCode) {
+      return isAnyOf(this.activityCode.activityCode, [
+        '1', '2', '4', '5',
```

Comment:
As this is the ```non-pass-finalisation``` page then we would never have a 1 i.e. a Pass

Also, we have an enum in the chat found here ```src/app/shared/models/activity-codes.ts```, therefore this should become

```
 return isAnyOf(this.activityCode.activityCode, [
         ActivityCodes.FAIL, 
         ActivityCodes.FAIL_PUBLIC_SAFETY,
         ActivityCodes.FAIL_CANDIDATE_STOPS_TEST,
]);
```